docs/SET_UP.md

```diff
@@ -0,0 +1,199 @@
+# Software Set Up
+This documentation includes the relevant information on how to set up the software frome scratch.
+
+The following software has been tested on Raspberry Pi OS (32-bit) bookworm with Python 3.11.2 and pip 23.0.1 
+
+
+## Raspberry Pi OS
+The Raspberry Pi OS can be installed on a usb stick or SD card by using the Raspberry Pi Imager as described [here](https://www.raspberrypi.com/software/).
+
+We recomend using Raspberry Pi OS (32-bit) bookworm as it the most recent version we have tested on.
+
+When switching on the Raspberry Pi for the first time there will be a short setup stage. For QUB students use the same password used for the other Raspberry Pi
+(if unsure ask a team member), for continuity across all our devices.
+
+## Raspberry Pi Configuration
+To be able to access some of the GPIO pins properly the Pi must be configured.
+
+First of all, if you haven't already connect the Raspberry pi to the internet.
+
+For QUB students see the image below. (This could take a few attempts)
+
+<img width="400" src="images/WiFi_screenshot.png">
+
+- Run the following
+```
+sudo raspi-config
+```
+- Select **3 Interface options**
+- Select **4 I2C**
+- Select **Yes**
+
+* Select **3 Interface options**
+*  Select **5 Serial port**
+*  Select **No**
+*  Select **Yes**
+
+- Select **3 Interface options**
+- Select **1 SSH**
+- Select **Yes**
+
+Next run
+```
+sudo nano /boot/firmware/config.txt
+```
+Add the following to the end of the file and save.
+```
+arm_64bit=0
+```
+Then reboot
+```
+sudo reboot
+```
+
+
+## git for collaborators
+This section is for collaborators when setting up a new Raspberry Pi. Open a new terminal and carry out the following.
+
+First the users name and email needs to be add this can be done by editing and running the following.
+```
+git config --global user.email "you@example.com"
+```
+```
+git config --global user.name "Your Name"
+```
+
+Next you need to specify how to reconcile divergent branches.
+```
+git config --global pull.rebase true
+```
+
+Next an SSH key needs to be generated and added to the ssh-agent and GitHub account.
+
+To generate an SSH key copy and paste the text below, substituting in your GitHub email address.
+
+```
+ssh-keygen -t ed25519 -C "your_email@example.com"
+```
+When prompted to "Enter a file in which to save the key", you can press Enter to accept the default file location.
+
+When prompted to "Enter passpharse", type the same passpharse used for the other devices on this project for continuity (if unsure ask a team member).
```

Comment:
I would advise against using a passphrase for the SSH key, otherwise you'll be prompted to enter your password every time you push